CHANGELOG.md

```diff
@@ -1,17 +1,16 @@
 # Alfred API - Changelog
 
-## 4.0.1 (yyyy-mm-dd)
+## 4.0.2 (yyyy-mm-dd)
```

Comment:
I'm considering making version nr of MVC release 4.1.0 to set it apart from the old Alfred API. If you want you can already set it here.

Reply:
tbf, I'd consider changing it to 5.0 since the installation requirements are changing?

Reply:
True, but it does not break any interface.
Anyway, let's keep that discussion out of scope for this PR.